app/src/main/java/com/google/samples/apps/nowinandroid/ui/NiaApp.kt

```diff
@@ -86,109 +87,119 @@ fun NiaApp(
         windowSizeClass = windowSizeClass
     ),
 ) {
-    val background: @Composable (@Composable () -> Unit) -> Unit =
-        when (appState.currentTopLevelDestination) {
-            TopLevelDestination.FOR_YOU -> {
-                content ->
-                NiaGradientBackground(content = content)
-            }
-            else -> { content -> NiaBackground(content = content) }
-        }
+    val showGradientBackground = appState.currentTopLevelDestination == TopLevelDestination.FOR_YOU
```

Comment:
nit: boolean variable names should be prefixed with `is|should|has`

Reply:
Updated!